sonar-jmeter-plugin/pom.xml

```diff
@@ -43,7 +43,7 @@
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-log4j12</artifactId>
       <version>${slf4j.version}</version>
-      <scope>test</scope>
+      <scope>provided</scope>
```

Comment:
maven build failed with compile errors, since SLF is used not only in testing code